docs/UserGuide.adoc

```diff
@@ -30,16 +30,21 @@ Interested? Jump to the <<Quick Start>> to get started. Enjoy!
 .  Ensure you have Java version `9` or later installed in your Computer.
 .  Download the latest `modsUni.jar` link:{repoURL}/releases[here].
 .  Copy the file to the folder you want to use as the home folder for modsUni.
-.  Double-click the file to start the app. The GUI should appear in a few seconds.
+.  Create a `data` folder in the home folder.
+.  Download the latest `modules.xml` link:{repoURL}/tree/master/data[here]
+.  Copy the XML file to the `data` folder.
+.  Double-click `modsUni.jar` to start the app. The GUI should appear in a few seconds.
 +
 image::Ui.png[width="790"]
-+
-.  Type the command in the command box and press kbd:[Enter] to execute it. +
-e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
-.  Some example commands you can try:
 
-* **`add`**`CS2103T` : adds the module CS2103T to the module configuration
-* **`remove`**`CS2103T` : removes the module CS2103T to the module configuration
+
+Type the command in the command box and press kbd:[Enter] to execute it. +
+e.g. typing *`help`* and pressing kbd:[Enter] will open the help window. +
+
+Some example commands you can try:
+
+* **`addModuleS`**`CS2103T` : adds the module CS2103T to the staged modules configuration
+* **`removeModuleS`**`CS2103T` : removes the module CS2103T to the staged module configuration
 * **`exit`** : exits the app
 
 .  Refer to <<Features>> for details of each command.
```

Comment:
I think this indentation isn't required. As mentioned in CS2101, if there is only 1 element the ordered list is not required.